### ISSUE TITLE
Use options instead of setters post construction where posssible

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/sun#325]
From  slack:

Chris Malley [11:55 AM] 
In NumberControl, is there a reason why the focus option is being set after instantiation, rather than in the options provided to constructor call?   Eg, for `leftArrowButton.focusable = false;`

Michael Kauzmann [11:55 AM] 
There is not!

Chris Malley [11:56 AM] 
Ditto for NumberSpinner.  Highly recommended to set options during construction. (edited)

Michael Kauzmann [11:56 AM] 
Thanks!

Jesse Greenberg [11:56 AM] 
Thanks, sounds good

Chris Malley [11:56 AM] 
`focus` is probably not an issue. but setting options after construction can result in additional work

new messages
[11:57] 
you also have (momentarily) an instance that is instantiated, but not in the state that you want it, something that is generally to be avoided. (edited)

Should be changed for NumberControl and NumberSpinner, but lets also comb through other places too and use options where possible.

**Comment by zepumph** [comment]
@mbarlow12 if you see any of these in places either, please make sure to change them!

**Comment by zepumph** [comment]
I'll  keep my eye open in the sims I'm working on. Unassigning for now.

**Comment by zepumph** [comment]
Here are some obvious ones that I found:
- [ ] ItemToolBoxNode 
- [ ] MagnetNodeWithField
- [ ] Book

**Comment by mbarlow12** [comment]
Note: https://github.com/phetsims/faradays-law/commit/9cdbedd3cd72e5c7c9108fb50a5ccbc3ccf9ad56 was performed in the keyboard_jump branch. Changes are identical, but it will lead to a merge conflict.

**Comment by mbarlow12** [comment]
@zepumph @jessegreenberg I searched through our sims for additional instances of the setter scenario and the usages are either within a listener function or used on a type that doesn't inherit from `Node` (e.g. `Screen.js`).

I think this can be closed but wanted to check with you two first.

**Comment by zepumph** [comment]
I looked through all usages of `.innerContent` and `.tagName` and only saw two files needing work. I feel good closing this, and updating any other usages as we stumble across them. Closing
